gateway/loaders/zookeeper/utils.go

```diff
@@ -16,7 +16,7 @@ type ZookeeperClient struct {
 	conn *zk.Conn
 }
 
-func (z *ZookeeperClient) GetZookeeperData() (*TargetConfig, error) {
+func (z *ZookeeperClient) GetZookeeperData(t *ZookeeperTargetLoader) (*TargetConfig, error) {
```

Comment:
it's used for logging so that it can use the logger specified in the gnmi-gw config
`t.config.Log.Error().Err(err).Msgf("Error unmarshalling into yaml for configuration at path: '%s'", crtPath)`

Reply:
sure